apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -116,7 +116,83 @@ test.describe("Send token (subAccount) - invalid input", () => {
   );
 });
 
-test.describe("Send token (subAccount) - valid input", () => {
+test.describe("Send token (subAccount) - invalid amount input", () => {
+  const tokenTransactionInvalid = [
+    {
+      transaction: new Transaction(Account.BSC_BUSD_1, Account.BSC_BUSD_2, "1", Fee.MEDIUM),
+      expectedWarningMessage: new RegExp(
+        /You need \d+\.\d+ BNB in your account to pay for transaction fees on the Binance Smart Chain network\. .*/,
+      ),
+    },
+    {
+      transaction: new Transaction(Account.ETH_USDT_2, Account.ETH_USDT_1, "1", Fee.MEDIUM),
+      expectedWarningMessage: new RegExp(
+        /You need \d+\.\d+ ETH in your account to pay for transaction fees on the Ethereum network\. .*/,
+      ),
+    },
+  ];
+  for (const transaction of tokenTransactionInvalid) {
+    test.use({
+      userdata: "speculos-2ETH-2BNB",
+      speculosCurrency: transaction.transaction.accountToDebit.currency,
+    });
+    test(
+      `Send from ${transaction.transaction.accountToDebit.accountName} to ${transaction.transaction.accountToCredit.accountName} - invalid amount input`,
+      {
+        annotation: {
+          type: "TMS",
+          description: "B2CQA-475",
+        },
+      },
+      async ({ app }) => {
+        await app.layout.goToAccounts();
+        await app.accounts.navigateToAccountByName(
+          transaction.transaction.accountToDebit.accountName,
+        );
+        await app.account.navigateToTokenInAccount(transaction.transaction.accountToDebit);
+        await app.account.clickSend();
+        await app.send.fillRecipient(transaction.transaction.accountToCredit.address);
+        await app.send.clickContinue();
+        await app.send.fillAmount(transaction.transaction.amount);
+        await app.send.checkContinueButtonDisabled();
+        await app.layout.checkWarningMessage(transaction.expectedWarningMessage);
+      },
+    );
+  }
+});
+
+test.describe("Send token (subAccount) - valid amount input", () => {
```

Comment:
I think this tests is duplicate of the next one

Reply:
Not exactly; one test is checking a valid amount, and the other one is checking a valid address. Since the address and the amount are not checked in the same modal, I needed to create two separate tests.

Reply:
why not combine those two into one test?